### ISSUE TITLE
Docs: typo/missing code for Texture drawing example

### ISSUE THREAD

**Issue by CrawfishPress** [kivy/kivy#7601]
Doc page:

https://kivy.org/doc/stable/api-kivy.graphics.texture.html

Code-file Docs:

kivy/graphics/texture.py

Text:

`# create a 64x64 texture, defaults to rgba / ubyte`
`texture = Texture.create(size=(64, 64))`
...
`# then, convert the array to a ubyte string`
`buf = b''.join(map(chr, buf))`

**Problem: This byte-string conversion isn't working in Python 3.8, gives error:**

 _TypeError: sequence item 0: expected a bytes-like object, str found_

**Fix: change code/doc to read:**

`# then, convert the array to a ubyte string`
`buf = ''.join(map(chr, buf)).encode('latin_1')`

The example section using Array.array copy, works fine. I don't know if this is a Python 3 problem, or what - looking at the docs, I actually thought the original line _should_ work as-is. But I couldn't get an example to work, kept getting a TypeError.

I found a StackOverflow question that appears to indicate it worked as of four years ago. [Texture Question](https://stackoverflow.com/questions/42226416/create-spectrum-using-kivy-texture)

Just noticed that the same docs occur in the Texture code-file, updated above accordingly.

**Comment by p0lygun** [comment]
@CrawfishPress 
instead of 
```
buf = b''.join(map(chr, buf))
```
we can simply do 
```
buf = bytes(buf)
```

**Comment by p0lygun** [comment]
documentation need an update, u can try to open a PR with relevant changes

**Comment by CrawfishPress** [comment]
`buf = bytes(buf)`

Well, if you want to do it the *easy* way, I guess that works also. :)

Re: the docs -  I looked at the docs website, but it's auto-generated from somewhere. I wasn't sure where, but when I went back and looked at my cloned Kivy repo, I couldn't even find the Python file `kivy/graphics/texture.py`, it was all `.pyd` files or something. So I'm not sure of the actual source of the docs (or code).
